sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -1581,7 +1844,8 @@ class StreamingOuterJoinSuite extends StreamingJoinSuite {
     }
   }
 
-  test("SPARK-49829 left-outer join, input being unmatched is between WM for late event and " +
+  testWithoutVirtualColumnFamilyJoins(
```

Comment:
Does this mean the bug has reoccured with virtual column family join? Could you please leave code comment about why we don't test with virtual column family? (Probably good to have a brief code comment for every single test which verifies with only one side (either with vcf or without vcf).

Reply:
Good idea, will add comments for these ones. For context, the ones marked with `testWithoutVirtualColumnFamilyJoins` at the moment all fail because they require `inferSchema` from StateDataSource.